src/components/App/App.tsx

```diff
@@ -0,0 +1,26 @@
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import Main from '../../pages/Main/Main';
+import CharacterDetails from '../../pages/CharacterDetails/CharacterDetails';
+import ErrorBoundary from '../ErrorBoundary/ErrorBoundary';
+import NotFound from '../../pages/NotFound/NotFound';
+
+const App: React.FC = () => {
+  return (
+    <ErrorBoundary>
+      <Router>
+        <Routes>
+          <Route path="/" element={<Main />}>
+            <Route
+              path="details"
+              element={<CharacterDetails searchTerm="" page={1} />}
```

Comment:
do you really need to use these props? If you need to have initial one, I guess you can define them just in CharacterDetails